src/Button/Button.js

```diff
@@ -84,6 +84,9 @@ export const styleSheet = createStyleSheet('Button', (theme) => {
         backgroundColor: palette.accent.A400,
       },
     },
+    raisedContrast: {
+      color: getContrastText(palette.primary[500]),
```

Comment:
I'm not sure why `getContrastText` is exported / imported. Can we stick to `palette.getContrastText`? So users can latter override the function. Thanks.